<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewClassWin</class>
 <widget class="QDialog" name="NewClassWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>590</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NEOLITH</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Creazione nuova classe</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="className">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NuovaClasse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Progetto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="projectName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>113</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Versione da aggiungere</string>
   </property>
  </widget>
  <widget class="QComboBox" name="versionAddTo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Percorso</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="creationPathEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="foldBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>112</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Eredita una classe</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="inheritGroup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Ereditariet√†</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>84</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome della classe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inClassName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>84</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File da includere</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inClassInclude">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&quot;file.h&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parte da ereditare</string>
    </property>
   </widget>
   <widget class="QComboBox" name="inScope">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Private</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Public</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Protected</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Descrizione</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="encapsuleGroup">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>290</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Incapsulamento</string>
   </property>
   <widget class="QCheckBox" name="encapPrivate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Incapsula private</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="encapPublic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Incapsula public</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="encapProtected">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Incapsula protected</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="constructorGroup">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Generazione costruttore</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>49</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Argomenti</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="constructorArgs">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>void</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Eredita</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="constructorInhe">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="infoUse">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>110</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Scrivi informazioni</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="generateDestructor">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>430</y>
     <width>117</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Genera distruttore</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="generateConstr">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>117</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Genera costruttore</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/icons/logics/Icons/class_icon.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="infoGroup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Informazioni</string>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Autore</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewClassWin</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewClassWin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
